src/onConsentChange.test.js

```diff
@@ -315,74 +183,22 @@ describe('under TCFv2', () => {
 
 		onConsentChange(callback);
 		invokeCallbacks();
+		await resolveAllPromises();
 
-		await waitForExpect(() => {
-			expect(callback).toHaveBeenCalledTimes(0);
-		});
+		expect(callback).toHaveBeenCalledTimes(0);
 
 		tcData.eventStatus = 'useractioncomplete';
 
 		invokeCallbacks();
+		await resolveAllPromises();
 
-		await waitForExpect(() => {
-			expect(callback).toHaveBeenCalledTimes(1);
-		});
+		expect(callback).toHaveBeenCalledTimes(1);
 
 		tcData.eventStatus = 'tcloaded';
 
 		invokeCallbacks();
+		await resolveAllPromises();
 
-		await waitForExpect(() => {
-			expect(callback).toHaveBeenCalledTimes(2);
-		});
-	});
-
-	it('callbacks executed in correct order', async () => {
```

Comment:
I notice this set of tests have been removed completely rather than adapted - is there really no way of testing this without waitForExpect?

Reply:
Added them back in. Thank you very much!